website/docs/language/resources/ephemeral/reference.mdx

```diff
@@ -13,14 +13,14 @@ This topic provides reference information for the `ephemeral` block.
 
 Ephemeral resources are Terraform resources that are essentially temporary. Ephemeral resources have a unique lifecycle, and Terraform does not store them in its state. Each `ephemeral` block describes one or more ephemeral resources, such as a temporary password or connection to another system.
 
-## Deferring
```

Comment:
Moving this below the dependency graph and giving it a longer name for more context